lib/src/over_react_redux/hooks/use_selector.dart

```diff
@@ -0,0 +1,183 @@
+///
+///     return Dom.div()('The current count is $count');
+///   },
+///   $CounterConfig, // ignore: undefined_identifier
+/// );
+/// ```
+///
+/// > Related: [useDispatch]
+TSelector useSelector<TSelector, TReduxState>(TSelector Function(TReduxState state) selector, [
+  bool Function(TSelector tNextSelector, TSelector tPrevSelector) equalityFn,
+]) {
+  ReactInteropValue jsSelector(ReactInteropValue jsState) => wrapInteropValue(selector(unwrapInteropValue(jsState)));
+  _JsReduxStateEqualityFn jsEqualityFn = equalityFn == null
+      ? null
+      : allowInterop((nextJsSelector, prevJsSelector) =>
+          equalityFn(unwrapInteropValue(nextJsSelector), unwrapInteropValue(prevJsSelector)));
+
+  return unwrapInteropValue<TSelector>(_jsUseSelector(allowInterop(jsSelector), jsEqualityFn));
+}
+
+@JS('ReactRedux.useSelector')
+external ReactInteropValue _jsUseSelector(_JsSelectorFn selector, [_JsReduxStateEqualityFn equalityFn]);
+
+// ----------------------------------------------------
+//  createSelectorHook
+// ----------------------------------------------------
+
+/// Returns a function that can be named and used as a replacement for [useSelector] when you want to
+/// use custom [context].
+///
+/// This is useful if you're building a complex reusable component, and you don't want your `Store` to collide
+/// with any Redux `Store` your consumers' applications might use.
+///
+/// See the [react-redux JS documentation](https://react-redux.js.org/api/hooks#custom-context) for more details.
+///
+/// ### Custom Context Example
+///
+/// ```dart
+/// import 'package:over_react/over_react.dart';
+/// import 'package:over_react/over_react_redux.dart';
+/// import 'package:redux/redux.dart';
+///
+/// // ------------------------------------
+/// //  1. Declare the custom context
+/// // ------------------------------------
+/// final MyContext = createContext<MyState>();
+/// final useMySelector = createSelectorHook(MyContext);
+/// final myStore = Store(myReducer);
+///
+/// // ------------------------------------
+/// //  2. (Optional) Create a custom
+/// //  provider component that uses the
+/// //  context defined above.
+/// // ------------------------------------
+/// mixin MyProviderProps = UiProps with ReduxProviderProps;
+///
+/// UiFactory<MyProviderProps> MyProvider = uiFunction(
+///   (props) {
+///     return (ReduxProvider()
+///       ..addUnconsumedProps(props, const [])
+///       ..context = MyContext
+///       ..store = myStore
+///     )(
+///       props.children,
+///     );
+///   },
+///   $MyProviderConfig, // ignore: undefined_identifier
+/// );
+///
+/// // ------------------------------------
+/// //  3. Create a function component that
+/// //  uses `useMySelector`.
+/// // ------------------------------------
+/// mixin MyComponentProps on UiProps {}
+///
+/// UiFactory<MyComponentProps> MyComponent = uiFunction(
+///   (props) {
+///     final count = useMySelector<int, MyState>((state) => state.count);
```

Comment:
I think this should be
```suggestion
///     final count = useMySelector((state) => state.count);
```